README.md

```diff
@@ -129,10 +129,34 @@ $ node src/app.js
 With the server running, open your favorite web browser and visit:
 http://localhost:3000
 
-If you want to specify the port number, set the `PORT` environment variable like so:
+### Environment Variables
+
+In order to mitigate uploading private keys, this repository uses a .env file for storing environment variables.
```

Comment:
Minor, but I don't think "mitigate" is the correct word here. The `dotenv` module is used to help _prevent_ committing sensitive configuration data to Github...and the reason the `.env` file does not get committed is that we have it in our `.gitignore` file. (If it weren't in `.gitignore`, contributors might accidentally commit it.)